extra/alacritty.man

```diff
@@ -42,8 +42,8 @@ Increases the level of verbosity (the max level is \fB\-vvv\fR)
 Prints version information
 .SH "OPTIONS"
 .TP
-\fB\-\-class\fR [ <instance> | <instance>,<general> ]
-Defines the window class hint on Linux [default: Alacritty,Alacritty ]
+\fB\-\-class\fR <instance> | <instance>,<general>
+Defines the window class hint on Linux [default: Alacritty,Alacritty]
```

Comment:
I feel like we should mention that on Wayland it sets `app_id` and in `--help` as well, since it's not something obvious and could make it clear for folks. Or even add `--app_id` cli argument(not sure that we really should though). What do you think?

Reply:
How does that work on Wayland? Only instance is supported and instance is app_id? General does nothing?

Reply:
I mean it's `app+id`, but yeah, only one thing exists, nothing more.